### PR TITLE
chore: added `nickname` to user edit field

### PR DESCRIPTION
**Pull request by Plebbimon**
also fixed `InternalGroupSelect` component

I'm not completely done here btw, need to put the `InputLeftLabel` component somewhere else.

### CODE REVIEW

**Comment by Plebbimon** [comment]
cHeCkmark*

File: src/modules/users/components/UserEditForm/useUserEditAPI.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export function useEditProfileAPI(user: UserNode) {
   const defaultValues = {
     firstName: user?.firstName ?? '',
     lastName: user?.lastName ?? '',
+    nickname: user?.nickname ?? 'drit',
```
Comment (alexaor) [inline src/modules/users/components/UserEditForm/useUserEditAPI.ts:30]: set fallback to blank string

File: src/modules/users/components/UserEditForm/useUserEditLogic.ts [hunk 2]
```diff
@@ -29,6 +30,7 @@ export type UserProfileCleanedData = Omit<
 const UserEditSchema = yup.object().shape({
   firstName: yup.string().required('Fornavn må fylles ut'),
   lastName: yup.string().required('Etternavn må fylles ut'),
+  nickname: yup.string().required('Kallenavn må fylles ut'),
```
Comment (alexaor) [inline src/modules/users/components/UserEditForm/useUserEditLogic.ts:33]: should this be mandatory?

**Review by alexaor — CHANGES_REQUESTED** [review]

File: src/modules/summaries/components/SummaryForm/SummaryForm.tsx [hunk 3]
```diff
@@ -55,58 +62,52 @@ export const SummaryForm: React.FC<SummaryFormProps> = ({
...
+                />
+              </CustomInputLabel>
+              <CustomInputLabel label={'Interngjeng'}>
                 <InternalGroupSelect
+                  withinPortal
                   withOtherOption
-                  setInternalGroupCallback={value =>
+                  internalGroupId={internalGroup}
+                  setInternalGroupCallback={value => {
+                    setInternalGroup(value)
```
[the remark this answers is not in the document]
Comment (Plebbimon) [reply src/modules/summaries/components/SummaryForm/SummaryForm.tsx:82]: okeydokey

File: src/modules/users/components/UserEditForm/useUserEditLogic.ts (hunk 2 above)
Comment (Plebbimon) [reply src/modules/users/components/UserEditForm/useUserEditLogic.ts:33]: No :)))

**Review by alexaor — APPROVED** [review]
LGTM

**Pull request merged**